### PR TITLE
Updates documentation landing page

### PR DESCRIPTION
**Pull request by bretterer**
## Description:
- Updates documentation landing page to match @alexdahl-okta PR okta/okta.github.io#2719

screencapture-localhost-8080-documentation-2019-04-01-17_02_39

### Resolves:

* closes okta/okta.github.io#2719

### CODE REVIEW

File: packages/@okta/vuepress-theme-default/assets/css/okta/pages/docsIndex.scss [hunk 1]
```diff
@@ -1,66 +1,98 @@
...
         }
       }
     }
   }
 
   &-content {
     .Row {
-      margin-left: -32px;
-      margin-right: -32px;
+      /*margin-left: -32px;
```
Comment (swiftone) [inline packages/@okta/vuepress-theme-default/assets/css/okta/pages/docsIndex.scss]: Still want this?

File: packages/@okta/vuepress-theme-default/components/GetStartedLinkGroup.vue [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
-      name: 'GetStartedLinkGroup',
-      props: ['section']
-    }
+  export default {
+    name: 'GetStartedLinkGroup',
+    props: ['section']
+  }
 </script>
+
+<style scoped>
```
Comment (swiftone) [inline packages/@okta/vuepress-theme-default/components/GetStartedLinkGroup.vue]: TIL

File: packages/@okta/vuepress-theme-default/assets/css/okta/pages/docsIndex.scss (hunk 1 above)
Comment (bretterer) [reply packages/@okta/vuepress-theme-default/assets/css/okta/pages/docsIndex.scss]: This is CSS from Alex, Going to leave it in there :)

**Comment by alexdahl-okta** [comment]
Screenshot looks good to me, and I _think_ my PR was in a reasonable state – not perfect, but I think working. I can review more deeply next week.

@nbarbettini should approve the content; I am happy to defer to him in the short-term. "How-to" is a stepping stone to guides, and could be replaced by guides when desired.

File: packages/@okta/vuepress-theme-default/components/GetStartedLinkGroup.vue (hunk 2 above)
Comment (nbarbettini) [reply packages/@okta/vuepress-theme-default/components/GetStartedLinkGroup.vue]: It's a nifty Vue feature!

**Pull request merged**